app/views/schools/placement_requests/acceptance/make_changes/new.html.erb

```diff
@@ -0,0 +1,47 @@
+<%= link_to 'Back', :back, class: 'govuk-back-link', data: {controller: 'back-link'} %>
+
+<%- self.page_title = "Confirm booking" %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-three-quarters">
+    <h1>
+      Confirm the booking for <%= @placement_request.candidate_name %> at <%= @placement_request.school.name %>
+    </h1>
+
+    <%= form_for @booking, url: schools_placement_request_acceptance_make_changes_path, method: 'post' do |f| %>
+      <%= GovukElementsErrorsHelper.error_summary @booking, 'There is a problem', '' %>
+      <div>
+
+
+        <% if @placement_request.school.availability_preference_fixed? %>
+          <h3 class="govuk-heading-m">The candidate requested</h3>
+
+          <p><%= @placement_request.requested_subject.name %> on <%= @placement_request.dates_requested %></p>
+        <% else %>
+          <h3 class="govuk-heading-m">The candidate requested</h3>
+
+          <p><%= @placement_request.requested_subject.name %></p>
+
+          <p class="govuk-inset-text"><%= @placement_request.dates_requested %></p>
+        <% end %>
+
+        <%= f.date_field :date, heading: true %>
```

Comment:
This screen is reached if the requested date is in the past. We don't explicitly say this to the user and probably should be calling out.